guides/welfare/sick_leave.md

```diff
@@ -6,18 +6,20 @@ Often when we think of sickness we think of colds, stomach bugs, and perhaps eve
 
 When people get sick we ask that they follow these guidelines:
 
-* Don't be a martyr. If you're sick, take the time off. We'd rather you stayed at home than came into the office and infected other people.
+* Don't be a martyr. If you're sick, take the time off. We'd rather you stayed at home than come into the office and infect other people.
 
-* We can provide support if you need help with your [state of mind](state_of_mind.md)
+* We can provide support if you need help with your [state of mind](state_of_mind.md).
 
-* If you need assistance with your sickness you can use our [employee assistance programme](employee_assistance.md)
+* If you need assistance with your sickness, you can use our [employee assistance programme](employee_assistance.md).
 
-* On your first day off, send an email to your team, or the operations team if you are on the Chalet, letting them know that you'll be away and what you had planned for that day. Aim to let them know as early as possible, so they can factor your absence into their tasks for the day.
+* On your first day off, let your team know (Slack, email, etc.), or the operations team if you are on the Chalet, that you'll be away and what you had planned for that day. Aim to let them know as early as possible, ideally by 10:30 AM, so they can factor your absence into their tasks for the day.
```

Comment:
Should this be start of business over 10.30am? I'd be worried if someone on my team didn't make contact until 10.30